content/fundamentals/lazy-loading-modules.md

```diff
@@ -6,6 +6,8 @@ Lazy loading can help decrease bootstrap time by loading only modules required b
 
 > info **Hint** If you're familiar with the **Angular** framework, you might have seen the "lazy-loading modules" term before. Be aware that this technique is **functionally different** in Nest and so think about this as an entirely different feature that shares similar naming conventions.
 
+> info **Info** Do note that [lifecycle hooks methods](https://docs.nestjs.com/fundamentals/lifecycle-events) are not called in lazy loaded modules and services.
```

Comment:
wouldn't it be better to merge these 2 hint / info-blocks? because otherwise 2 blocks would directly appear after each other?!

Reply:
IMHO this should be turned into a warning (with yellow background and Warning prefix)